<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptoconTargetLinuxView</class>
 <widget class="QMainWindow" name="OptoconTargetLinuxView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>961</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Channels</string>
    </property>
    <widget class="QTextEdit" name="textEdit_Limit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>2.00</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CheckLimit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>check limit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_Set">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>941</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="checkBox_A1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A1</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_A1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_A2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_A2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A2</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_A3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_A3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A3</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_A4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>120</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_A4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A4</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_selectAll">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>select all</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionRefresh_Connection"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh_Connection">
   <property name="text">
    <string>Refresh Connection</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
